modules/virt-vm-custom-scheduler.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-schedule-vms.adoc      
+
+:_content-type: PROCEDURE                                             
+[id="virt-vm-custom-scheduler_{context}"]                                   
+= Scheduling virtual machines with a custom scheduler
+
+You can use a custom scheduler to schedule a virtual machine (VM) on a node.
+
+.Prerequisites
+* A secondary scheduler is configured for your cluster.
+
+.Procedure
+
+. Add the custom scheduler to the VM configuration by editing the `VirtualMachine` manifest. For example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    spec:
+      schedulerName: my-scheduler <1>
+      domain:
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          rng: {}
+        resources:
+          requests:
+            memory: 1Gi
+      terminationGracePeriodSeconds: 180
+# ...
+----
+<1> The name of the custom scheduler. If the `schedulerName` value does not match an existing scheduler, the `virt-launcher` pod stays ini `Pending` state until the specified scheduler is found.
+
+. Verify that the VM is using the custom scheduler by checking the `virt-lanuncher` pod events. For example:
++
+[source,terminal]
+----
+$ oc describe pod virt-launcher-vm-fedora-dpc87
+----
++
+.Example output
+[source,terminal]
+----
+[...] 
+Events:
+  Type    Reason     Age   From              Message
+  ----    ------     ----  ----              -------
+  Normal  Scheduled  21m   my-scheduler  Successfully assigned default/virt-launcher-vm-fedora-dpc87 to node01
+[...]
```

Comment:
Maybe an additional phrase explaining that `my-scheduler` is the desired output? Something like:
Note that the pod has been scheduled using `my-scheduler` which is the scheduler name provided in the VM definition.

Reply:
I reworded the introductory sentence for the `Verification` section to include this info.

Reply:
I still feel that readers may not find obvious that they should look for the `From` field in pod description. What do you think about something like:

```
  Type    Reason     Age   From              Message
  ----    ------     ----  ----              -------
  Normal  Scheduled  21m   my-scheduler <2> Successfully assigned default/virt-launcher-vm-fedora-dpc87 to node01
```

<2> Name of the configured scheduler in the VM manifest. Check the scheduler name is matching the custom scheduler name provided in the VM manifest.

Reply:
Per our style guidelines, I cannot add a numbered callout at that location, but I have added a lead-in sentence explaining the output. Let me know if that works.

Reply:
Preview build: https://62977--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-schedule-vms.html#virt-vm-custom-scheduler_virt-schedule-vms

Reply:
Works perfectly :)